packages/design-docs/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@talend/design-docs",
+  "version": "1.3.0",
+  "description": "Package containing design.talend.com stories",
+  "homepage": "https://github.com/Talend/ui#readme",
+  "main": "src/index.ts",
+  "scripts": {
+    "build-storybook": "echtalend-scripts build-storybook --docs",
```

Is it a typo _echtalend_? 
```suggestion
    "build-storybook": "talend-scripts build-storybook --docs",
```